src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const BASIC_COST_PER_DAY = days * 40;
+
+  const SHORT_RENTAL_TIME = 3;
+  const DISCOUNT_SHORT_TIME = 20;
+
+  const LONG_RENTAL_TIME = 7;
+  const DISCOUNT_LONG_TIME = 50;
+
+  if (days >= LONG_RENTAL_TIME) {
+    return BASIC_COST_PER_DAY - DISCOUNT_LONG_TIME;
+  }
+
+  if (days >= SHORT_RENTAL_TIME) {
+    return BASIC_COST_PER_DAY - DISCOUNT_SHORT_TIME;
+  }
+
+  return BASIC_COST_PER_DAY;
```

Comment:
Prefer using 'const' over 'let' for the 'BASIC_COST_PER_DAY' variable since its value is not being reassigned. This will ensure the variable is not unintentionally modified elsewhere in the code.